### DISCUSSION TITLE
What principle is used to disable the quick feedback of win+ F to do this?

### DISCUSSION THREAD

**Discussion by Natural-selection1** [valinet/ExplorerPatcher#2735] [Q&A] [answered]
Because Microsoft uses the very nice win button (its location) to take up some very convenient shortcuts, I want to disable things like clipboard (win V), search (win S), etc., so that I can set my own software shortcuts to win + ....

**Comment by pyrates999** [comment, marked as answer]
https://www.howtogeek.com/282080/how-to-disable-the-built-in-windows-key-shortcuts/

**Reply by Natural-selection1** [reply]
Thanks, bro.
